<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Путь:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="currentPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="copyPath">
       <property name="text">
        <string>копировать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2" colspan="2">
      <widget class="QComboBox" name="comboBoxType"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fileName"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="open">
       <property name="text">
        <string>открыть / создать</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Имя файла:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTreeView" name="treeView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
